### DISCUSSION TITLE
开帖子反馈问题规则

### DISCUSSION THREAD

**Discussion by MaxMKMax** [getlantern/lantern-forum-cn#978] [Announcements]
大家好. 如果有问题请单独开GitHub帖子, 别在别人的帖子底下留言, 避免我们漏掉您的问题. 

也请别直接给我发邮箱, 除非我通知您了. 因为我工作的时间有限, 无法查看每个邮箱. 不好意思啊. 麻烦你们了. 谢谢配合. Cheers! 

谢谢各位支持Lantern, 感激不尽!

**Comment by Coasenorth** [comment]
@Mike777Khoo 您好，我的邮箱是[htifly@126.com](mailto:htifly@126.com)，请帮忙查一下。[
![1265361704](https://github.com/user-attachments/assets/8d4ac1bb-9f67-48c5-859f-735f9429654d)
](url)
续费两年，始终没有更新。我8月份就续费了，到现在都没有更新。

**Comment by MaxMKMax** [comment]
@Coasenorth 您好. 好的, 已经帮您提交上去解决问题了.

**Reply by Coasenorth** [reply]
续费记录已经更新了，谢谢。

**Comment by MaxMKMax** [comment]
@Coasenorth 您好. 不客气. 谢谢您支持蓝灯&相信我们!

**Comment by garyguan1989** [comment]
请看我@你的  349002735@qq.com   一直无法连接·申请退款

**Reply by garyguan1989** [reply]
![373919610-69edc28b-05da-41ba-8fde-1ea21c03cc61](https://github.com/user-attachments/assets/439be78d-05ba-46b9-9afa-053fbb5a8e23)

**Comment by MaxMKMax** [comment]
@garyguan1989 您好. 请问您愿意等我们一下吗? 我们会尽快修复好. 请您先别退款. 感激不尽! 谢谢!

**Reply by garyguan1989** [reply]
已经半年了·一直不退款

**Comment by guofulibai123** [comment]


**Reply by guofulibai123** [reply]
@Mike777Khoo 帮忙处理一下谢谢

**Reply by wumingdaxia2** [reply]
请问余额宝支付渠道怎么找到的，我怎么没有

**Comment by wumingdaxia2** [comment]
您能给个付款链接吗，我只有支付宝和微信支付。
用电脑和手机都无法付款。电脑板只有信用卡选项，手机版支付宝点不开。
![截图未命名1102203525](https://github.com/user-attachments/assets/7e8851df-1616-4a68-96c2-6f98a9a7dd55)
![截图未命名1102203555](https://github.com/user-attachments/assets/1ca8bd6b-0308-4033-83c6-7bdd7bdd2697)

**Comment by MaxMKMax** [comment]
@wumingdaxia2 您好. 请您提供我您的账号邮箱. 我给您去拿个支付链接. 谢谢!

**Comment by garyguan1989** [comment]
我的支付宝是349002735@qq.com 请尽快退款

**Reply by MaxMKMax** [reply]
@garyguan1989 您好. 请您试试看和支付系统或者银行退款看可以吗?

**Reply by garyguan1989** [reply]
什么时候退款，半年了

**Comment by fenghuanggongjiang** [comment]
您好，我购买专业版半年了，但是现在还是登录不上，显示是免费版，采用邮箱登录找回验证码，但是显示我的邮箱不对，请我帮我确认什么问题，我的邮箱是330896993@qq.com ，谢谢

**Comment by XDDDDDDDLYB** [comment]
![_cgi-bin_mmwebwx-bin_webwxgetmsgimg__ MsgID=635308873491287194 skey=@crypt_61f32618_31892379d65253d80c9ffe124250a4cb mmweb_appid=wx_webfilehelper](https://github.com/user-attachments/assets/9f84b029-5496-4409-8bdc-e56d625e9736)
您好，我想申请退款
